src/sections/Contributors/Contributors.css

```diff
@@ -1,9 +1,9 @@
 .con-body {
 	color: var(--primaryYellow);
-	height: 200vh;
+	/* height: 200vh;
```

Comment:
Please delete unnecessary comments

Reply:
Deleted the unnecessary comments along with unused styles